tests/testing_data/yml_training_files/chat_client_config.yml

```diff
@@ -0,0 +1,32 @@
+config:
+  api_server_host_url: http://testserver
```

Comment:
change to the real chat client config as seen in template/chat_client_config.yml